ghcide.cabal

```diff
@@ -127,6 +134,7 @@ library
         Development.IDE.LSP.Outline
         Development.IDE.LSP.Protocol
         Development.IDE.LSP.Server
+        Development.IDE.Session
```

Comment:
What about only expose that module if the flag has the correct value (only inside the appropiate branch)?

Reply:
That's probably a better idea than all the CPP! Also I think it might be failing to build on window ghc-8.10: https://github.com/bubba/haskell-language-server/runs/873687931?check_suite_focus=true
Maybe because its generating an empty module?

Reply:
I've changed this but looks like the PR isn't picking up on this force push...